app/helpers/application_helper.rb

```diff
@@ -1,6 +1,8 @@
 module ApplicationHelper
   APP_HOST = ENV['APP_HOST']
   APP_HOST_LEGACY = ENV['APP_HOST_LEGACY']
+  REGEXP_REPLACE_TRAILING_EXTENSION = /\.(\w+)$/
+  REGEXP_REPLACE_WORD_SEPARATOR = /-|_/
```

Comment:
On freeze toutes les nouvelles constantes (en attendant de freezer toutes celles existantes)

```suggestion
  REGEXP_REPLACE_TRAILING_EXTENSION = /\.(\w+)$/.freeze
  REGEXP_REPLACE_WORD_SEPARATOR = /-|_/.freeze
```